Makefile

```diff
@@ -72,8 +72,13 @@ test-codespell:
 
 .PHONY: test-flake8
 test-flake8:
+	echo "\033[0;31mWARNING\033[0m: Did you mean to run \`make ruff\`?"
 	flake8 $(SOURCES)
 
+.PHONY: test-ruff
+test-ruff:
+	ruff .
```

Comment:
maybe `ruff $(SOURCES)` instead? To avoid looking at virtual env files, etc

Reply:
The default `exclude` excludes .git, venv, build, etc. ([Full default config](https://github.com/charliermarsh/ruff#configuration)), so this ends up scanning the same files. I'll update it to use `$(SOURCES)` for consistency though.